desafio-8/desafio-8.css

```diff
@@ -0,0 +1,80 @@
+header {
+ padding: 20px 0;
+ background-color: #9932CC;
+ color: #ffffff;
+ display: flex;
+ align-items: center;
+ justify-content: center;
+ flex-direction: row;
+}
+
+body{
+ background-color:#222222;
+ font-family: Inter;
+}
+
+main {
+ width: 940px;
+ margin: 0 auto;
+}
+
+form {
+ margin: 40px 0;
+ display: flex;
+ justify-content: center;
+ flex-direction: column;
+ align-items: center;
+ color: #ffffff;
+ gap: 20px;
+}
+
+.input-padrao {
```

Comment:
why did you use this class `padrao` ? 

I'm asking because all classes are in English

Reply:
I changed it to the English default